Representable/representable
# non-approved cois don't show on state map page

**changes**
- makes it so that all entries with admin_approved field set to False will not show up on the state map page
- cois with admin_approved set to false submitted to a drive will still show up (with hidden tag) to org admins on drive map pages

**to test**
- draw coi and set admin approved to false
- check state map page to see that coi doesn't appear

**note**
- this branch should only be merged following #758 and following an alteration of the prod db to make all current communities have admin_approved set to True, since the current site automatically sets this field to false for all communities drawn without following a drive link
